crates/autopilot/src/boundary/order.rs

```diff
@@ -37,3 +37,45 @@ pub fn to_domain(
         signature: order.signature.into(),
     }
 }
+
+/// Recover order uid from order data and signature
+pub fn order_uid(
```

Comment:
Can we move this method into the domain module? This would require a lot less publicity.

Reply:
moved to tokenized